### DISCUSSION TITLE
项目使用GPU运行，为什么问问题时CPU会升高？

### DISCUSSION THREAD

**Discussion by xxxuexx** [chatchat-space/Langchain-Chatchat#2897] [General]
'gptq_wbits': 16,
 'gpus': '0,1',
 'infer_turbo': False,
 'limit_worker_concurrency': 10,
 'max_gpu_memory': '10GiB',
当前Embbedings模型： text2vec-multilingual @ cuda

项目使用GPU运行，为什么每次问问题时CPU会升高？，查看是C:\Users\admin\.conda\envs\Langchain-Chatchat\python.exe 导致cpu升高，为什么会这样？不应该走GPU运算吗？

**Comment by zRzRzRzRzRzRzR** [comment]
是应该走GPU运算，，升高了多少
